my-app/src/API/index.ts

```diff
@@ -0,0 +1,8 @@
+export const API_KEY: string = 'bb2b538ec7dc87a9a9ecaa9065c3baff';
+
+export const getListUrl = (lng: string, list: string): string =>
```

Comment:
Better to use full names without shorcuts

Reply:
done